.github/workflows/cd.yml

```diff
@@ -10,58 +10,58 @@ jobs:
           ref: ${{ github.ref }}
-      - name: Get Kibana version
-        id: kbn_version
+      - name: Get OpenSearch Dashboards version
+        id: osd_version
         run: |
-          echo "::set-output name=kbn_version::$(jq -r '.kibana.version' ./kibana/plugins/opendistro_security/package.json)"
+          echo "::set-output name=osd_version::$(jq -r '.opensearchDashboards.version' ./OpenSearch-Dashboards/plugins/security-dashboards-plugin/package.json)"
       - name: Get plugin version
         id: plugin_version
         run: |
-          echo "::set-output name=plugin_version::$(jq -r .version ./kibana/plugins/opendistro_security/kibana.json)"
-      - name: Use kibana release commit
+          echo "::set-output name=plugin_version::$(jq -r .version ./OpenSearch-Dashboards/plugins/security-dashboards-plugin/opensearch_dashboards.json)"
+      - name: Use OpenSearch Dashboards release commit
         run: |
-          cd ./kibana
-          git checkout tags/v${{ steps.kbn_version.outputs.kbn_version }} -b v${{ steps.kbn_version.outputs.kbn_version }}
+          cd ./OpenSearch-Dashboards
+          git checkout tags/v${{ steps.osd_version.outputs.osd_version }} -b v${{ steps.osd_version.outputs.osd_version }}
       - name: Get node and yarn versions
         id: versions
         run: |
-          echo "::set-output name=node_version::$(cat ./kibana/.node-version)"
-          echo "::set-output name=yarn_version::$(jq -r '.engines.yarn' ./kibana/package.json)"
+          echo "::set-output name=node_version::$(cat ./OpenSearch-Dashboards/.node-version)"
+          echo "::set-output name=yarn_version::$(jq -r '.engines.yarn' ./OpenSearch-Dashboards/package.json)"
       - name: Setup node
         uses: actions/setup-node@v1
         with:
           node-version: ${{ steps.versions.outputs.node_version }}
           registry-url: 'https://registry.npmjs.org'
-      - name: Install correct yarn version for Kibana
+      - name: Install correct yarn version for OpenSearch Dashboards
         run: |
           npm uninstall -g yarn
           echo "Installing yarn ${{ steps.versions_step.outputs.yarn_version }}"
           npm i -g yarn@${{ steps.versions.outputs.yarn_version }}
-      - name: Bootstrap Kibana
+      - name: Bootstrap OpenSearch Dashboards
         run: |
-          cd ./kibana
-          yarn kbn bootstrap --oss
+          cd ./OpenSearch-Dashboards
+          yarn osd bootstrap --oss
       - name: build security plugin
         id: build_zip
         run: |
-          cd ./kibana/plugins/opendistro_security
-          yarn build --kibana-version ${{ steps.kbn_version.outputs.kbn_version }}
-          artifact_path=`ls $(pwd)/build/opendistroSecurity*.zip`
+          cd ./OpenSearch-Dashboards/plugins/security-dashboards-plugin
+          yarn build --opensearch-dashboards-version ${{ steps.kbn_version.outputs.kbn_version }}
+          artifact_path=`ls $(pwd)/build/opendistro_security*.zip`
```

Comment:
Security plugin has been renamed to `opensearch-security`.

Reply:
more detail updated in this PR context